From 764c1750c18fc3fe4005fcb5b912ce9e39bc2b7f Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Mon, 23 Mar 2020 21:51:31 +0000
Subject: [PATCH] configure.ac: remove duplicate AC_CONFIG_MACRO_DIR

Autoconf 2.70 will fatally error out if AC_CONFIG_MACRO_DIR is called more than once:

| configure.ac:25: error: AC_CONFIG_MACRO_DIR can only be used once
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 913500a..1d174c5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,8 +22,6 @@ AC_SUBST(LT_CURRENT)
 AC_SUBST(LT_REVISION)
 AC_SUBST(LT_AGE)
 
-AC_CONFIG_MACRO_DIR([m4])
-
 DOXYGEN_REQ_VER=1.8.8
 
 dnl Checks for programs.
